stencil-library/src/index.html

```diff
@@ -4,144 +4,21 @@
 <head>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
-  <title>Stencil Component Starter</title>
-
-  <script type="module" src="/build/webcomponents.esm.js"></script>
-  <script nomodule src="/build/webcomponents.js"></script>
+  <title>JustiFi Component Examples</title>
 </head>
 
 <body>
-  <h1>Card Form</h1>
-  <hr>
-  <justifi-card-form style-overrides='{
```

Comment:
This file was broken up into multiple `example.html` files. It probably should have been it's own PR, which can still be done if desired by reviewer.